node/rpc/Cargo.toml

```diff
@@ -15,6 +15,8 @@ serde_json = "1.0"
 jsonrpc-core = "15.0.0"
 jsonrpc-pubsub = "15.0.0"
 
+fc-rpc-core = "0.1.0"
```

Comment:
If there is a good reason to rename packages, we should use the same strategy for all of them - and provide git and branch properties.

Reply:
Also this crate should be coming from github like our other frontier crates.